scripts/run_backend_tests.py

```diff
@@ -261,9 +337,17 @@ def main(args=None):
 
         os.environ['PYTHONPATH'] = os.pathsep.join(pythonpath_components)
 
-    if parsed_args.test_target and parsed_args.test_path:
+    test_specs_provided = sum([
+        1 if argument else 0
+        for argument in (
+            parsed_args.test_target, parsed_args.test_path,
+            parsed_args.test_shard)
```

Comment:
optional: len(set(...)) > 2 (Though this one is also not very redable or easy to understand)

Reply:
I think the existing code is more readable than relying on the fact that duplicate `None` values will be removed in a set